platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_util.py

```diff
@@ -340,6 +347,73 @@ def system_ready():
         return False
     return True
 
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+PLATFORM_API2_WHL_FILE_PY2 ='sonic_platform-1.0-py2-none-any.whl'
+PLATFORM_API2_WHL_FILE_PY3 ='sonic_platform-1.0-py3-none-any.whl'
```

Comment:
All applications which import sonic_platform should now support Python 3 (as of a couple weeks ago). Therefore, I believe there should no longer be a need to build/install the Python 2 package. Would you mind confirming, and if so, remove all references to the Python 2 package in this PR?